src/blockchain/Admins.js

```diff
@@ -627,7 +628,7 @@ class Admins {
         return service.patch(pledgeAdmin.typeId, {
           status: 'Canceled',
           mined: true,
-        });
+        }, { eventTxHash: event.transactionHash });
```

Comment:
why use eventTxHash? We have always used txHash in the past.

Edit: I think I see, this is never saved, but is used in hooks to only execute notification / conversation logic when the event has been mined. Is that correct?

Reply:
exactly

Reply:
It's just a param forwarded to the patch hook